tests/SimpleStringTest.cpp

```diff
@@ -480,6 +480,13 @@ TEST(SimpleString, Characters)
     CHECK(s == s2);
 }
 
+TEST(SimpleString, CharactersAsDecimals)
+{
+    SimpleString s(DecimalStringFrom(-5));
+    SimpleString s2(DecimalStringFrom(-5));
+    CHECK(s == s2);
```

Comment:
In my opinion, we would do well to change all of them to something like:
```c++
SimpleString s(DecimalStringFrom(-5);
STRCMP_QUAL("-5", s.asCharString());
```
Any reason why these tests were written like this?